XML/TestCases/StressTests.xml

```diff
@@ -181,4 +181,37 @@
 		<Tags>stress,hv_storvsc,storage</Tags>
 		<Priority></Priority>
 	</test>
+	<test>
+		<testName>STRESS-STORAGE-4K-IO-WITHVERIFY-LONGTIME</testName>
+		<testScript>PERF-STORAGE-MULTIDISK-RAID0-FIO.ps1</testScript>
+		<setupType>OneVM12Disk</setupType>
+		<AdditionalHWConfig>
+			<DiskType>Managed</DiskType>
+			<StorageAccountType>PERF_STORAGE_ACCOUNT_TYPE</StorageAccountType>
+		</AdditionalHWConfig>
+		<setupScript>.\Testscripts\Windows\SETUP-Check-PowerPlan.ps1,.\Testscripts\Windows\AddHardDisk.ps1</setupScript>
+		<files>.\Testscripts\Linux\utils.sh,.\Testscripts\Linux\perf_fio.sh,.\Testscripts\Linux\fio_jason_parser.sh,.\Testscripts\Linux\JSON.awk,.\Tools\gawk</files>
+		<TestParameters>
+			<param>modes=STRESS-STORAGE-4K-IO-MODES</param>
+			<param>chunkSize=256</param>
+			<param>ioSize=5000T</param>
+			<param>fileSystem=xfs</param>
+			<param>startQDepth=64</param>
+			<param>maxQDepth=2048</param>
+			<param>startIO=4</param>
+			<param>maxIO=256</param>
+			<param>fileSize=16G</param>
+			<param>ioruntime=FIO_STRESS_LONGTIME_RUNTIME</param>
+			<param>PHYSICAL_NUMBER=1</param>
+			<param>SCSI=1,1,RAID</param>
+			<param>parseTimeout=FIO_LOG_PARSE_TIMEOUT</param>
+			<param>testType=StressLongTime</param>
+		</TestParameters>
+		<cleanupScript>.\Testscripts\Windows\RemoveHardDisk.ps1</cleanupScript>
+		<Platform>Azure,HyperV</Platform>
+		<Category></Category>
```

Comment:
Missing Category, Area & Priority

Reply:
Here we have a pipeline to trigger this test.
https://microsoft.visualstudio.com/LSG/_apps/hub/ms.vss-ciworkflow.build-ci-hub?_a=edit-build-definition&id=51315
In this pipeline, we will lock the resource group before starting the long stress test and unlock it when it is done.

Reply:
We don't need Category, Area and Priority. It is triggered just by the pipeline.

Reply:
We need those information in LISAv2.

Reply:
Some pipelines may trigger LISAv2 tests with Category, Area or Priority. If I set those information, this test may be triggered by other pipelines which is not expected. This test is only triggered by https://microsoft.visualstudio.com/LSG/_apps/hub/ms.vss-ciworkflow.build-ci-hub?_a=edit-build-definition&id=51315
So I think it's  better for not setting those information just the same with the test case "STRESS-STORAGE-4K-IO-WITHVERIFY".

Reply:
Then suggest new value. I will look into